src/hotspot/share/code/codeCache.cpp

```diff
@@ -1559,6 +1559,34 @@ void CodeCache::log_state(outputStream* st) {
             unallocated_capacity());
 }
 
+void CodeCache::write_perf_map(outputStream* st) {
+  MutexLocker mu(CodeCache_lock, Mutex::_no_safepoint_check_flag);
+
+  char fname[32];
+  jio_snprintf(fname, sizeof(fname), "/tmp/perf-%d.map", os::current_process_id());
+
+  fileStream fs(fname, "w");
+  if (!fs.is_open()) {
+    log_warning(codecache)("Failed to create %s for perf map", fname);
+    st->print_cr("Failed to create %s", fname);
+    return;
+  }
+
+  AllCodeBlobsIterator iter(AllCodeBlobsIterator::only_alive_and_not_unloading);
+  while (iter.next()) {
+    CodeBlob *cb = iter.method();
+    ResourceMark rm;
+    const char* method_name =
+      cb->is_compiled() ? cb->as_compiled_method()->method()->external_name()
+                        : cb->name();
+    fs.print_cr(INTPTR_FORMAT " " INTPTR_FORMAT " %s",
```

Comment:
Indentation isn't right.

Reply:
Do you mean how the arguments are lined up on the continuation line below?